docs/source/release-notes/workarea-3-5-0.html.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Workarea 3.5.0 Release Notes
+excerpt: Release notes for Workarea 3.5.0
+---
+
+Workarea 3.5.0 Release Notes
+======================================================================
+
+The following changes are introduced in Workarea 3.5.0.
+
+_See also:_ [Workarea 3.5.0 Upgrade Guide](/upgrade-guides/workarea-3-5-0.html) and [Workarea Upgrade](https://plugins.workarea.com/plugins/upgrade) (tool you should use for all upgrades).
+
+
+Releases
+----------------------------------------------------------------------
+
+* Improves release previewing by including all releases scheduled to publish before the previewed release
+* Removes release undo functionality in favor of using separate "undo" releases, and adds a workflow for creating them
+* Restores the releases calendar feature in the Admin
+* Allows data file importing in releases
+* Allows featured product changes in releases
```

Comment:
(minor) This may be nitpick, but featured product changes in releases were always available, the changes to the search index allow accurate previewing of this now.